index.html

```diff
@@ -27,7 +27,7 @@
       </div>
   </header>
   <!--"main" container for all the sections of the card-->  
-  <main id="portfolio" >
+  <main id="portfolio">
```

Comment:
- [ ] The mobile version seems to be broken after the implementation of the desktop version. There is a whitespace to the right and some cards are going extremely out to the right. Please look at demo of your project to understand what I am trying to say here.

Test Demo|
:-------------:|
![a1](https://user-images.githubusercontent.com/39523312/159917851-a9294590-65c1-4e37-aa42-ec21fa6503ce.gif)